GLOCKFILE

```diff
@@ -20,7 +20,8 @@ github.com/cockroachdb/c-lz4 c40aaae2fc50293eb8750b34632bc3efe813e23f
 github.com/cockroachdb/c-protobuf 4feb192131ea08dfbd7253a00868ad69cbb61b81
 github.com/cockroachdb/c-rocksdb b7fb7bddcb55be35eacdf67e9e2c931083ce02c4
 github.com/cockroachdb/c-snappy 5c6d0932e0adaffce4bfca7bdf2ac37f79952ccf
-github.com/cockroachdb/stress 574c7f17016a4db745b88f6643700995110bdd07 
+github.com/cockroachdb/pq 77bd85500f4521560720328957bae47a78a90ed1
+github.com/cockroachdb/stress 574c7f17016a4db745b88f6643700995110bdd07
```

Comment:
Appears to be just some sort of weird error in the diff computation. I don't think there are any whitespace differences. And the file was generated via `glock save`.